### PR TITLE
[OBMIN-323] Product post data

### PR DESCRIPTION
**Pull request by Voloshkina**
Created ProductPostData component

### CODE REVIEW

**Review by ASGladchenko — APPROVED** [review]

**Review by Kyslytsia — APPROVED** [review]

**Review by Mish-hash — APPROVED** [review]

**Review by AkintsevaElizaveta — APPROVED** [review]

**Review by Viktoria-Serkiz — APPROVED** [review]

File: src/components/product-post-data/styles.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import styled, { css } from 'styled-components';
+
+export const Container = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  padding-bottom: 40px;
+  height: 100%;
+  background-color: ${({ theme }) => theme.colors.colorPopup};
```
Comment (cloudmenson) [inline src/components/product-post-data/styles.ts:9]: When you have multiple callbacks, you can make one

**Review by cloudmenson — APPROVED** [review]

File: src/components/product-post-data/ProductPostData.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  component: ProductPostData,
+  argTypes,
+} as ComponentMeta<typeof ProductPostData>;
+
+const Template: ComponentStory<typeof ProductPostData> = (args) => (
+  <OwnerAndPost>
+    <ProductPostData {...args} />
+  </OwnerAndPost>
+);
+export const ProductPostDefault = Template.bind({});
```
Comment (Wolshebnik) [inline src/components/product-post-data/ProductPostData.stories.tsx:18]: space

File: src/components/product-post-data/styles.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  color: ${({ theme }) => theme.colors.colorGrey};
+`;
+
+export const PostDataDescriptionOl = styled.ol`
+  margin-left: 10px !important;
+  padding-right: 20px;
+`;
+
+export const PostDataDescriptionOlItem = styled.li`
+  font-family: Proxima Nova, sans-serif;
```
Comment (Wolshebnik) [inline src/components/product-post-data/styles.ts:51]: remove this, we don't have it

File: src/components/product-post-data/ProductPostData.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  title: 'pages/Product Post Data',
+  component: ProductPostData,
+  argTypes,
+} as ComponentMeta<typeof ProductPostData>;
+
+const Template: ComponentStory<typeof ProductPostData> = (args) => (
+  <OwnerAndPost>
+    <ProductPostData {...args} />
+  </OwnerAndPost>
+);
```
Comment (Wolshebnik) [inline src/components/product-post-data/ProductPostData.stories.tsx:17]: space

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**